sparkle/src/components/Breadcrumbs.tsx

```diff
@@ -56,24 +63,24 @@ export function Breadcrumbs({ items }: BreadcrumbProps) {
           >
             <Icon visual={item.icon} className="s-text-brand" />
             {item.label === ELLIPSIS_STRING ? (
-              <DropdownMenu>
-                <DropdownMenu.Button>{ELLIPSIS_STRING}</DropdownMenu.Button>
-                <DropdownMenu.Items origin="topLeft">
-                  {itemsHidden.map((item, index) => (
-                    <DropdownMenu.Item
-                      icon={item.icon}
-                      label={item.label}
-                      link={item.href ? { href: item.href } : undefined}
-                      key={`breadcrumbs-hidden-${index}`}
-                    >
-                      {truncateWithTooltip(
-                        item.label,
-                        LABEL_TRUNCATE_LENGTH_MIDDLE
-                      )}
-                    </DropdownMenu.Item>
-                  ))}
-                </DropdownMenu.Items>
-              </DropdownMenu>
+              <NewDropdownMenu>
+                <NewDropdownMenuTrigger asChild>
+                  <Button variant="ghost" label={ELLIPSIS_STRING} />
+                </NewDropdownMenuTrigger>
+
+                <NewDropdownMenuContent align="start" className="min-w-[220px]">
```

Comment:
Do we really need the arbitrary value, here?

Reply:
Not really 👍🏼